toolchain/papiTest.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2018 IBM.
+# Author: Umesh S <umess1@linux.vnet.ibm.com>
+
+
+import os
+from avocado.utils import process
+from avocado import Test
+from avocado import main
+from avocado.utils import git, build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class papiTest(Test):
```

Comment:
As per Python standard, better no to use camel case for Class name. Change to something like `Papitest`